src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardHelper.js

```diff
@@ -1,1058 +1,611 @@
+        return sortedActiveFields;
+    },
 
-            // BatchInfo module public functions and properties
-            return {
-                // record info
-                name: '',
-                id: '',
-                description: '',
-                expectedCount: 0,
-                expectedTotal: 0,
-                recordCount: 0,
-
-                // batch processing settings
-                requireTotalMatch: false,
-                batchProcessSize: 0,
-                runOpportunityRollupsWhileProcessing: false,
-                donationMatchingBehavior: '',
-                donationMatchingClass: '',
-                donationMatchingOptions: [],
-                donationMatchingRule: [],
-                donationDateRange: '',
-                postProcessClass: '',
-                load: load,
-                isValid: isValid,
-                noMatchOnDate: false,
-                onInfoUpdated: _onInfoUpdated
-            }
-        })(this.Event());
+    /**
+     * @description Gets the active fields sorted and grouped by SObject.
+     * @return Map of SObject group to List of related active fields.
+     */
+    getActivesBySObject: function (component) {
+        let activeFields = this.getActives(component);
+        var activesBySObject = this.groupFieldsBySObject(activeFields);
+        return activesBySObject;
     },
 
     /**
-     * @description Gets the Batch Fields module.
-     * @return Model module of the Batch Fields.
+     * @description Sort the fields by order.
+     * @param fields. List of the fields to sort.
+     * @return sorted fields.
      */
-    BatchFields : function() {
-        return (function (Event) {
-            var _allFields = [];
-            var _onFieldsUpdated = new Event(this);
-
-            /* ******************PUBLIC FUNCTIONS*************************/
-
-            /**
-             * @description Load the fields and notify onFieldsUpdated listeners.
-             * @param allFields: list of allFields with sObjectLabel/Name.
-             * param activeFields: Map of activeFieldsBySObject with sObjectName, Name,
-             * and Default Value, Hide and Required flags.
-             * @return void.
-             */
-            function load(allFields, activeFields) {
-                _allFields = [];
-                var activeFieldMap = new Map();
-
-                if (activeFields) {
-                    activeFields.forEach(function(activeField) {
-                        var fieldId = activeField.sObjectName + '.' + activeField.name;
-                        activeFieldMap.set(fieldId, activeField);
-                    });
-                }
+    sortFieldsByOrder: function (fields) {
+        fields.sort(function (currentField, nextField) {
+            if (currentField.sortOrder < nextField.sortOrder) {
+                return -1;
+            }
+            if (currentField.sortOrder > nextField.sortOrder) {
```

Comment:
Is there a standard argument for or against using `else if` in this situation? I get that they're functionally the same, just curious.